aiida/backends/djsite/settings.py

```diff
@@ -13,23 +13,13 @@
 from __future__ import division
 from __future__ import print_function
 
-import sys
-
-import os
 from sqlalchemy.dialects.postgresql import JSONB
 from sqlalchemy.dialects.postgresql import UUID
 
 from aiida.common import exceptions
 from aiida.common.timezone import get_current_timezone
 from aiida.manage.configuration import get_profile, settings
 
-# Assumes that parent directory of aiida is root for
-# things like templates/, SQL/ etc.  If not, change what follows...
```

Comment:
Not sure what exactly this comment means / why this should be needed.
Anyhow, it seems the code works also without this